AcceptanceTests/AcceptanceTests.Common/Driver/Helpers/WaitHelper.cs

```diff
@@ -12,7 +12,11 @@ class WaitHelper
     {
         public WebDriverWait newWait(NgWebDriver Driver)
         {
-            WebDriverWait wait = new WebDriverWait(Driver, timeout: TimeSpan.FromSeconds(30))
+            return newWait(Driver, 30);
+        }
```

Comment:
If this method is just going to call the other with 30 could we just make that an optional argument instead?